spanner/spansql/parser.go

```diff
@@ -1236,6 +1245,15 @@ func (p *parser) parseAlterTable() (*AlterTable, *parseError) {
 			return a, nil
 		}
 
+		if p.eat("ROW") {
```

Comment:
Suggestion: Would it maybe make more sense to check if you can eat `ROW DELETION POLICY` here, and then only parse the actual expression. The same would then also apply to the `REPLACE ROW DELETION POLICY`?

Reply:
Thanks!

Certainly, this makes easier to switch if another syntax is added after the ROW token.

I fixed at d83fb538d9bd852a452a8db1811612b11eefac43